src/dlgAboutDialog.cpp

```diff
@@ -1062,3 +1070,18 @@ void dlgAboutDialog::setSupportersTab(const QString& htmlHead)
     textBrowser_supporters->setDocument(supportersDocument.get());
     textBrowser_supporters->setOpenExternalLinks(true);
 }
+
+QString dlgAboutDialog::createBuildInfo() const {
+
+    QString buildInfo = tr("<table border=\"0\" style=\"margin-bottom:36px; margin-left:36px; margin-right:36px;\" width=\"100%\" cellspacing=\"2\" cellpadding=\"0\">\n"
+                           "<tr><td colspan=\"2\" style=\"font-style:italic;\">It would be useful to copy and include information below when reporting an issue!<br></td></tr>");
+
+    buildInfo.append(tr("<tr><td style=\"font-weight: 800\">Build information:</td><td>Build: %1</td></tr>").arg(mudlet::self()->version));
+    buildInfo.append(tr("<tr><td></td><td>OS: %1</td></tr>").arg(QSysInfo::prettyProductName()));
+    buildInfo.append(tr("<tr><td></td><td>CPU Architecture: %1</td></tr>").arg(QSysInfo::currentCpuArchitecture()));
+    buildInfo.append(tr("<tr><td></td><td>Branch: %1</td></tr>").arg(GIT_BRANCH));
```

Comment:
When embedding `#define`d values into the source code I've found that `QByteArray(GIT_BRANCH)` works better that including the raw string literal or `QString(GIT_BRANCH)` though `QStringLiteral(GIT_BRANCH)` would be an option.  We do not really want raw strings being converted into `QString`s because it means that other coders cannot tell whether they are supposed to EITHER be translated (for presentation to the end-user as part of the UI) by being wrapped in `QObjectLLtr(...)` or `QApplication::translate(...)` OR used internally as `QStringLiteral(...)`s or `QLatin1String(...)`s.  Though we are not likely to throw the big switches that `define`-ing `QT_NO_CAST_FROM_ASCII` and `QT_NO_CAST_TO_ASCII` represent - one of those will cause the compiler to error out at every one of the places where raw string literals (as GIT_BRANCH will be) are converted to a `QString` without a wrapper....

tl;dr; I'd like to sugesst
```suggestion
    buildInfo.append(tr("<tr><td></td><td>Branch: %1</td></tr>").arg(QByteArray(GIT_BRANCH)));
```